htdocs/product/stock/replenish.php

```diff
@@ -84,6 +84,10 @@
 if ($count == 1 && (empty($fk_entrepot) || $fk_entrepot <= 0) && getDolGlobalString('MULTICOMPANY_PRODUCT_SHARING_ENABLED')) {
 	$fk_entrepot = $lastWarehouseID;
 }
+//If the warehouse is set to the default selected user
+if ((empty($fk_entrepot) || $fk_entrepot <= 0) && getDolGlobalString('MAIN_DEFAULT_WAREHOUSE_USER')) {
```

Comment:
Does this means the use can't set warehouse to another value?
Mays be it is better to do 
`if (!GETPOSTISSET('fk_warehouse') && (empty($fk_entrepot) || $fk_entrepot <= 0) && getDolGlobalString('MAIN_DEFAULT_WAREHOUSE_USER')) {`
So user can still change value if default is not the good one ?

Reply:
@eldy it's done